packages/cysync-core/src/context/buySell/index.tsx

```diff
@@ -194,44 +195,54 @@ export const BuySellProvider: FC<BuySellContextProviderProps> = ({
     setUnhandledError(e);
   }, []);
 
+  const onRetry = useCallback(() => {
+    setState(BuySellState.CURRENCY_SELECT);
+    resetUserInput();
+  }, []);
```

Comment:
Are all errors supposed to retry the flow from the start?

Reply:
Will pick in a different PR